components/automate-ui/src/app/page-components/simple-line-graph/simple-line-graph.component.scss

```diff
@@ -0,0 +1,191 @@
+@import "~styles/variables";
+
+:host {
+  display: block;
+  width: 100%;
+  margin-bottom: 16px;
+
+  .label-container {
+    width: 100%;
+    height: 20px;
+    margin-top: -20px;
+    padding: 1px;
+    background: $chef-white;
+  }
+}
+
+::ng-deep {
+  svg {
+    background: $chef-white;
+    padding: .5em;
+  }
+
+  path,
+  line,
+  .tick,
+  .grid {
+    stroke: $eggplant-light-1;
+    stroke-width: 1px;
+  }
+
+  .line {
+    fill: transparent;
+    stroke: black;
+    stroke-width: 1px;
+  }
+
+  .y-axis {
+    transform: translateX(20px);
+  }
+
+  .x-axis {
+    .tick line {
+      display: none;
+    }
+  }
+
+  circle.point {
+    fill: $eggplant-dark-2;
+  }
+
+  .ring {
+    position: absolute;
+    stroke-width: 1;
+    fill: none;
+    stroke: url(#chef-gradient-purple);
+
+    opacity: 0;
+    visibility: hidden;
+    pointer-events: none;
+    transition: .5s opacity;
+
+    &.active {
+      opacity: 1;
+      visibility: visible;
+    }
+  }
+
+  .graph-button {
+    display: flex;
+    position: absolute;
+    z-index: 5;
+    background: transparent;
+    padding: 1px;
+    border: none;
+    border-radius: 4px;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    transition: transform 1s;
+
+    &.active {
+      background: #3023AE;
+      background: linear-gradient(315deg, #3023AE, #C86DD7);
```

Comment:
Should make the colors named constants, particularly since you reuse them several times.